src/pages/Home.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import { useNavigate } from 'react-router-dom';
+import { AiOutlineArrowRight } from 'react-icons/ai';
+
+const Home = () => {
+  const data = useSelector((state) => state.rootReducer.data);
+  const navigate = useNavigate();
+
+  const handleItemClick = (date) => {
+    const filteredData = data.filter((item) => item.date === date);
+    navigate('/details', { state: { data: filteredData } });
+  };
+
+  return (
+    <div className="main-container">
+      <div className="image-container">
+        <h2 className="image-title">Company Information</h2>
+      </div>
+      <h3 className="stats-by-year">Stats By Year</h3>
+      <div className="stats-container">
+        {data.map((item, index) => (
+          <div className={index === 1 || index === 2 ? 'item-card even' : 'item-card odd'} key={item.date}>
+            <button className="my-button" type="button" onClick={() => handleItemClick(item.date)}>
+              <AiOutlineArrowRight />
+            </button>
```

Comment:
As part of the project [requirements](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md#:~:text=When%20the%20page%20loads%2C%20the%20webapp%20shows%20the%20list%20of%20categories%20that%20could%20be%20filtered%20out%20by%20some%20parameter%2C%20for%20example%20by%20the%20category%20name.), the homepage should have a way of filtering through the fetched data and i believe using a searching feature would be a great idea. Consider this in order to satisfy the project requirements.

Reply:
Hi @DuaneDave I am lost on this one, I am filtering the data by years, selecting which data I want to show after that. How would the filter needs to be constructed? Your answer would be appreciated since I want to apply this as soon as possible.

Reply:
Hi, @edgardsoza as I understand what the reviewer requested, you're not filtering the `data` to change the content of the home screen 🏠  you're just filtering it to the `data` to navigate to the details screen 💻 , consider adding a search bar, or a list of dates to pick from to actually change the data in the main screen to pick from. 😉